src/user_management/user.model.js

```diff
@@ -23,17 +23,15 @@ export default (sequelize, DataTypes) => {
       type: DataTypes.STRING,
       allowNull: false
     }
-  },
-    {
-      instanceMethods: {
-        hashPassword = (password) => {
-          return bcrypt.hash(password, bcrypt.genSaltSync(10));
-        },
-        validatePassword = (password) => {
-          return bcrypt.compare(password, this.password);
-        }
-      }
-    })
+  })
+
+  User.prototype.validPassword = function (password) {
+    return bcrypt.compareSync(password, this.password)
```

Comment:
- Don't ever do anything synchronously on a NodeJS server. You'll be blocking all requests until this finishes 😄
- You're missing the `salt` for the password comparison, which must be the same one used when encrypting the original password.